pytwis/pytwis_clt.py

```diff
@@ -246,13 +253,13 @@ def pytwis_command_parser(raw_command):
         arg_dict = parse.parse('{old_password} {new_password} {confirmed_new_password}', args)
         if arg_dict is None:
             raise ValueError('{} has incorrect arguments'.format(CmdConstant.CMD_CHANGE_PASSWORD))
-        elif arg_dict['new_password'] != arg_dict['confirmed_new_password']:
+        elif arg_dict[CmdConstant.ARG_NEW_PASSWORD] != arg_dict['confirmed_new_password']:
```

Comment:
Define an ARG constant for 'confirmed_new_password'?

Reply:
Fixed